ubid.rb

```diff
@@ -68,6 +68,7 @@ class UBID
   TYPE_FIREWALL_RULE = "fr"
   TYPE_FIREWALL = "fw"
   TYPE_POSTGRES_FIREWALL_RULE = "pf"
+  TYPE_USAGE_ALERT = "va"
```

Comment:
Should we be making ubid types for every kind of alert or should there be TYPE_ALERT and it's polymorphic in some other way?

Reply:
I decided to use UBID::TYPE_ETC